python/lsst/ip/isr/deferredCharge.py

```diff
@@ -744,21 +744,30 @@ class DeferredChargeCalib(IsrCalib):
     serialTraps : `dict` [`str`, `lsst.ip.isr.SerialTrap`]
         A dictionary, keyed by amplifier name, containing a single
         serial trap for each amplifier.
+
+    Also, the values contained in this calibration are all derived
+    from and image and overscan in units of electron as these are
+    the most natural units in which to compute deferred charge.
+    However, this means the the user should supply a reliable set
+    of gains when computing the CTI statistics during ISR.
+
+    Version 1.1 deprecates the USEGAINS attribute and standardizes
+        everything to electron units.
     """
     _OBSTYPE = 'CTI'
     _SCHEMA = 'Deferred Charge'
-    _VERSION = 1.0
+    _VERSION = 1.1
 
-    def __init__(self, useGains=True, **kwargs):
+    def __init__(self, **kwargs):
         self.driftScale = {}
         self.decayTime = {}
         self.globalCti = {}
         self.serialTraps = {}
 
```

Comment:
I think you need a check here to see if `useGains` is in the `kwargs` dict, and if so (a) pop it out, and (b) give a warning that it is deprecated.  Something like
```
if kwargs.pop("useGains", None) is not None:
    warnings.warn("useGains is deprecated, and will be removed after vXX.", FutureWarning)
```
(and make sure that we have `import warnings` at the top)